examples/basic/object_detection_image.ipynb

```diff
@@ -77,88 +78,38 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "ad4f4a5e",
-   "metadata": {
-    "scrolled": true,
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "# list all AI models available for inference\n",
-    "zoo.list_models()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "dc2baec4",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "# load AI model 'ssd_mobilenet_v2' for DeGirum Orca AI accelerator\n",
-    "# (change model name to \"...n2x_cpu_1\" to run it on CPU)\n",
-    "model = zoo.load_model(\"mobilenet_v2_ssd_coco--300x300_quant_n2x_orca1_1\")"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "27299231",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "# configure some result visualization parameters (optional step)\n",
-    "model.image_backend = \"pil\" # use PIL as image rendering backend\n",
-    "model.overlay_show_probabilities = True # show class probabilities on overlay image\n",
-    "model.overlay_font_scale = 1.5 # increase font size for overlay image labels\n",
-    "model.overlay_alpha = 1 # set minimum transparency for overlay image labels"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "03954556",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "# perform AI model inference on given image file (assuming the current directory is where this notebook is located)\n",
-    "res = model(\"../../images/TwoCats.jpg\")"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "bb207347",
+   "id": "fea1e8c8",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
+    "import degirum as dg, degirum_tools\n",
+    "# configure for Google Colab\n",
+    "degirum_tools.configure_colab() \n",
```

Comment:
Just as a note. When I see this google Colab call, it unnerves me. As a DeGirum developer, I know that this is necessary to get the samples to work in Colab. But I think that an external developer who's not using Colab would be similarly unnerved. What's this mysterious call do? Is Colab required? What is Colab, even? All of this is on the second line and distracts from PySDK. We should at least write a much clearer comment. The current one adds nothing.
This concern is not so acute after this refactor, as non-notebook samples exist and don't have this.